javatrix/Matrix.java

```diff
@@ -141,6 +141,29 @@ public Matrix(int m, int n, double s) throws IllegalArgumentException
         }
     }
     /**
+     * identity -Constructs an identity matrix with the specified
+     *  dimensions.
+     * @param m -The number of rows in the identity matrix.
+     * @param n -The number of columns in the identity matrix.
+     * @return identityMatrix -The (m x n) identity matrix. 
+    **/
+    public static Matrix identity(int m, int n) 

```

Comment:
What happens if we are provided with invalid verticies (negative, I guess)? Also, aren't identity matricies all square? Why should it be possible to create a rectangular one? I'm not sure it's worth worrying about the rectangular ones since the method signature came from the API docs that we were given, but I do think we need to throw an IllegalArgument exception on negative input and write a couple of tests then ensure that it happens.